devicetypes/smartthings/zwave-radiator-thermostat.src/zwave-radiator-thermostat.groovy

```diff
@@ -198,8 +217,12 @@ def off() {
 }
 
 def setHeatingSetpoint(setpoint) {
+	if (isPoppRadiatorThermostat()) {
+		sendEvent(name: "heatingSetpoint", value: setpoint, unit: temperatureScale)
```

Comment:
I understand why you did this, since I bet the app will just spin forever unless you do, but I'm just going to note that it seems wrong to do this when we know that it won't actually be set for up to 4 hours.

Reply:
Yeah, I know, but since we don't have any 'pending' indicator in OneApp or something else that would inform user that it still hasn't been change I think it's lesser evil.